deploy/gvisor/Dockerfile

```diff
@@ -12,9 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:18.04
-RUN apt-get update && \
-    apt-get install -y kmod gcc wget xz-utils libc6-dev bc libelf-dev bison flex openssl libssl-dev libidn2-0 sudo libcap2 && \
-    rm -rf /var/lib/apt/lists/*
+# Need an image with chroot
+FROM alpine:3
```

Comment:
I suggest pinning this to a more specific version to avoid future headaches.
alpine:3.10.3 list of versions can be found here https://hub.docker.com/_/alpine

Reply:
Nice this is way better!

Reply:
Honestly the likelihood of `chroot` failing is pretty low. I'd rather not have to micro-manage patch versions if I can. If this wider pin proves a problem, I'll pin this more tightly.